src/styles/blocks/button.scss

```
@@ -0,0 +1,33 @@
+.button {
+  box-sizing: border-box;
+  margin: 0 auto;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: $c-general;
+  width: 288px;
+  height: 56px;
+  font-weight: 700;
+  color: $c-white;
+  text-decoration: none;
+  text-align: center;
+  border: 3px solid transparent;
+  border-radius: 25px;
+
+  cursor: pointer;
+
+  &--primary {
+    width: 100%;
+  }
```

```suggestion
  &--fullwidth {
    width: 100%;
  }
```

By default `primary` modifier is used for color.